CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 ## 6.5.8 - upcoming release
```

Comment:
This was already released. Should be 6.6.0 i think because is adding something new.

Reply:
@digna-ionos done in 0dd524f63097523725cc3bd727cceedc5b6bc9f9